cmd/server/wire_gen.go

```diff
@@ -0,0 +1,28 @@
+// Code generated by Wire. DO NOT EDIT.
```

Comment:
Duplicated InitServer function in wire.go and wire_gen.go

Reply:
see here: https://github.com/google/wire/tree/master/_tutorial